src/constants.ts

```diff
@@ -19,6 +19,5 @@
  * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
+// eslint-disable-next-line import/prefer-default-export
 export const IDENTITY_PROVIDER_INRUPT_PROD = "https://broker.pod.inrupt.com";
```

Comment:
this made more sense to me than switching to default export and switching back when we add another constant, but happy to amend